components/CardStack.jsx

```diff
@@ -0,0 +1,127 @@
+import {
+  motion,
+  useMotionValue,
+  useTransform,
+  useMotionTemplate,
+} from "framer-motion";
+import React, { useState } from "react";
+import Card from "../components/Card";
+import inactiveCards from "../data/inactiveCards";
+
+export const CardStack = ({ vocabularies: vocabulariesprop }) => {
+  const [vocabularies, setVocabularies] = useState(vocabulariesprop);
```

Comment:
Thinking about it, it might make sense to name the prop `initialVocabularies` to best describe what it actually does.

```suggestion
export const CardStack = ({ initialVocabularies }) => {
  const [vocabularies, setVocabularies] = useState(initialVocabularies);
```